webapp/crashstats/tokens/jinja2/tokens/home.html

```diff
@@ -115,7 +115,7 @@ <h2>Generate a New Token</h2>
         <form method="post">
           {% csrf_token %}
           <table class="data-table">
-            {{ form }}
+            {{ form.as_table() }}
```

Comment:
Django project created a new div form renderer that's accessibility friendly which the table form renderer is not. Calling `{{ form }}` will change what it's doing from the table form renderer it does now to the div form renderer. So we call `form.as_table()` here because I didn't want to spend time on redoing the API token form using the div form renderer and hunt for other forms that need to be similarly updated.

I'll write up a bug to cover that work.

http://docs.djangoproject.com/en/5.0/releases/4.1/

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1883044